viral_load_plot.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+from matplotlib import pyplot as plt
+from utils import _sample_viral_load
+
+"""Samples the viral_load function 100 times and output a chart of the viral load over num_days"""
+NUM_DAYS = 11
+viral_loads = []
+x = np.linspace(1, NUM_DAYS, NUM_DAYS)
+
+for i in range(100):
+    vals = _sample_viral_load().pdf(x)
+    viral_loads.append(vals)
+viral_loads = np.array(viral_loads)
+
+print(f"viral_loads means: {viral_loads.mean(axis=0)}")
+print(f"viral_loads stds: {viral_loads.std(axis=0)}")
+
+fig, ax = plt.subplots(1, 1)
+ax.errorbar(x, viral_loads.mean(axis=0), yerr=viral_loads.std(axis=0), lw=2, label='new noisy gamma pdf')
+ax.plot(x, [0.05, 0.1, 0.2, 0.3, 0.25, 0.2, 0.15, 0.1, 0.05, 0., 0.], lw=2, label='old viral load dist')
+plt.legend()
+plt.xlabel("Days since infection")
+plt.ylabel("Viral load")
+plt.title("Viral Load (Noisy Gamma Model)")
+plt.savefig("plots/viral_load.png")
```

Comment:
Can we put helper files/plotting files/notebook in a separate folder?  I think Nasim has been using `scratch`, but feel free to use anything else if that's more appropriate. People can put their analysis in that folder moving forward.

Reply:
ok -- I'll try to figure out how to add it. I was having some path issues when I tried to put it in a sub-folder originally, and didn't know how to fix it without restructuring the project

Reply:
sgtm